Example/ConsentViewController_ExampleTests/Extensions/GDPRWebViewExtensionsSpec.swift

```diff
@@ -44,24 +49,25 @@ class GDPRWebViewExtensionsSpec: QuickSpec {
                     webview.getAuthId { result, _ in authId = result }
                 }
                 webview.loadHTMLString("", baseURL: URL(string: "https://sourcepoint.com")!)
-
                 expect(authId).toEventually(equal("foo"))
             }
         }
 
         describe("setAuthId") {
+            beforeSuite {
```

Comment:
Since you already changed the timeout in the top most `beforeSuite` you don't need to do it again in the same file.

Reply:
@andresilveirah does this change affects only this suite or the whole set of suites it belongs to? Do I need to set it back to default value after this suite is done?

Reply:
That's a good question @Nevazhnovu . According to [Quick's documentation](http://quick.github.io/Quick/Functions.html#/s:5Quick11beforeSuiteyyyycF), that block of code will run only once per suite of specs (e.g. per file). However, since [`AsyncDefaults`](https://quick.github.io/Nimble/Structs/AsyncDefaults.html#/s:6Nimble13AsyncDefaultsV7timeout8Dispatch0E12TimeIntervalOvpZ) is a global, I'd say it's of good practice to add a:
```swift
afterSuite {
    AsyncDefaults.timeout = _default value_
}
```

Reply:
Done